modules/nvidia_plugin/tests/unit/test_networks.hpp

```diff
@@ -27,7 +27,7 @@ inline std::shared_ptr<ngraph::Function> CreateMatMulTestNetwork() {
 
 class SuperDummyOp : public ov::op::Op {
 public:
-    inline static constexpr type_info_t type_info{"SuperOperation", 0ul};
```

Comment:
I think in order to avoid error you can move this operation to `OPENVINO_OP` macro

Reply:
This change doesn't depend on the changes in OpenVINO

Reply:
Why cannot you replate this variable to `OPENVINO_OP` macro?